BUILD.bazel

```
@@ -0,0 +1,56 @@
+licenses(["notice"])
+
+exports_files(["LICENSE"])
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "magic_enum",
+    hdrs = ["include/magic_enum.hpp"],
+    strip_include_prefix = "include",
```

Have you tested actually consuming this as an external dependency? I ask because IIRC `strip_include_prefix` doesn’t work in that case because there will be an extra `external/` to contend with. I have not tested that your change _doesn’t_ work, but the below modification definitely does.

```suggestion
    includes = ["include"],
```

---

Yes, that's much nicer. Thanks for the suggestion, done.